src/rdb.c

```diff
@@ -1819,6 +1875,35 @@ robj *rdbLoadObject(int rdbtype, rio *rdb, sds key, int dbid, int *error) {
                 }
             }
 
+            /* This will also be called when the set was just converted
+             * to a listpack encoded set. */
+            if (o->encoding == OBJ_ENCODING_LISTPACK) {
+                if (setTypeSize(o) < server.set_max_listpack_entries &&
+                    sdslen(sdsele) <= server.set_max_listpack_value &&
+                    lpSafeToAdd(o->ptr, sdslen(sdsele)))
+                {
+                    unsigned char *p = lpFirst(o->ptr);
+                    if (p && lpFind(o->ptr, p, (unsigned char*)sdsele,
+                                    sdslen(sdsele), 0))
+                    {
```

Comment:
I'll unwrap this.

Because you're asking: Yes, I use large font and a half-laptop wide editor and in other projects I like to keep the limit at 80.

![Screenshot from 2022-10-13 08-29-04](https://user-images.githubusercontent.com/273886/195519037-123b63b1-8d5d-4b6a-a4e9-34e181571f0f.png)